about.html

```diff
@@ -101,6 +119,169 @@
 <section>
     <div class="about-block">
         <div class="container">
+            <img src="assets/community.svg" class="img-responsive">
+            <h2>Community</h2>
+            <p class="supportparagraph hidden-xs hidden-sm">Our vibrant community empowers us to create an amazing platform. Dynamic developers, cutting edge scientists as well as everyday users work together to further Jupyter's best-in-class tools. Here is a dynamic visualization of our most recent contributors:</p>
+            <p class="supportparagraph hidden-md hidden-lg">Our vibrant community empowers us to create an amazing platform. Dynamic developers, cutting edge scientists as well as everyday users work together to further Jupyter's best-in-class tools.</p>
```

Comment:
Just make "Here is a dynamic visualization of our most recent contributors:" a separate p,that is hidden on smapp devices.